node/forcedroid.js

```diff
@@ -42,7 +42,7 @@ var fs = require('fs');
 var cordovaHelper = require('../external/shared/node/cordovaHelper');
 var miscUtils = require('../external/shared/node/utils');
 
-var version = '3.1.0';
```

Comment:
This is only the ```npm``` package version number. We could do a force override, but that's not preferable (I don't know what will happen to the users who have already downloaded ```3.1.0```).